### DISCUSSION TITLE
救命，nonebot不支持arm64的嘛

### DISCUSSION THREAD

**Discussion by 932220954** [nonebot/discussions#208] [问答]
<img width="570" alt="image" src="https://user-images.githubusercontent.com/14266822/190458912-74d71aaf-8223-497f-9a40-f45875e38dd6.png">

这怎么搞，大佬求助

**Comment by yanyongyu** [comment]
m1的mac？盲猜你pydantic装错了版本或者你Python装错了架构，你可以尝试重装一下pydantic，确认一下是不是arm版的

**Comment by mnixry** [comment]
看起来像是Python装成了x86的版本，然后靠着Rosetta运行起来了
